lib/telegraph/account.rb

```diff
@@ -0,0 +1,20 @@
+module Telegraph
+  class Account
+    attr_reader :access_token, :short_name, :author_name, :auth_url, :author_url, :page_count
+
+    def initialize(attrs)
```

Comment:
I believe something like `dry-struct` (or ol' good virtus) would be helpful here, for initialize API objects with keyword arguments and provide them as a fields.